virtualenvwrapper.el

```diff
@@ -532,6 +539,7 @@ virtualenvwrapper.el."
   ;; make completions work
   (venv--make-pcompletions ("workon" "rmvirtualenv"
                             "cdvirtualenv" "cpvirtualenv"))
+  (add-hook 'venv-postactivate-hook 'venv--check-virtual-proj-path)
```

Comment:
I'd rather leave hooks for user code and not use them for our own stuff. Would you just call `venv--check-virtual-project-path` [here](https://github.com/porterjamesj/virtualenvwrapper.el/blob/master/virtualenvwrapper.el#L230) in `venv--activate-dir` directly, rather than relying on the hook.